ouroboros-consensus/src/Ouroboros/Consensus/Util.hs

```diff
@@ -83,6 +91,7 @@ import qualified Data.Set as Set
 import           Data.Void
 import           Data.Word (Word64)
 import           GHC.Stack
+import qualified "canonical-json" Text.JSON.Canonical as CJSON
```

Comment:
I was surprised to see `-XPackageImports`. Is that because of `hackage-security`? We probably don't need to defend against that conflict. Am I missing another risk?

Reply:
removed